src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String generateRandomColor() {
+        int rand = new Random().nextInt(5);
```

Comment:
new Random should be class-level variable, don't need to create it each time you call generateRandomColor method